README.md

```diff
@@ -25,32 +25,76 @@ make
 ```
 
 ## Usage
+```
+go get github.com/nats-io/gnatsd
+gnatsd
+MF_AGENT_BOOTSTRAP_ID=11:22:33:44 MF_AGENT_BOOTSTRAP_KEY=5341a625-53a5-4a25-be74-5b648243fcbc MF_AGENT_NATS_URL=nats://localhost:4223 MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap build/mainflux-agent
```

Comment:
This looks specific to your setup. I would prefer.
```
MF_AGENT_BOOTSTRAP_ID=<agent_bootstrap_id> \
MF_AGENT_BOOTSTRAP_KEY=<agent_bootstrap_key> \
... 
``` 
btw, this is normal? `nats://localhost:4223`

Reply:
@manuio `4223` is default NATS port, but I think this one is not needed, as service could use this default by itself.

Reply:
@drasko default NATS port is 4222.